docs/UserGuide.md

```diff
@@ -345,16 +348,16 @@ Allows you to delete a medical condition from a specified patient in the system.
 * Only one medical condition can be deleted at a time.
 
 **Examples**:
-* `list_p` followed by `delete_tag_p 2 t\diabetes` deletes the medical condition `diabetes` of the 2nd patient in the patient list.
-* `find_p Jonathan` followed by `delete_tag_p 1 t\cancer` deletes the medical condition `cancer` of the 1st patient in the results of the `find_p` command.
+* Entering `list_p` followed by `delete_tag_p 2 t\diabetes` deletes the medical condition `diabetes` of the 2nd patient in the patient list.
+* Entering `find_p Jonathan` followed by `delete_tag_p 1 t\cancer` deletes the medical condition `cancer` of the 1st patient in the results of the `find_p` command.
 
 Back to [Patient Management Features](#patient-management-features).
 
 ### Editing the remark of a patient : `remark_p`
 
 Allows you to edit the remark of a specified patient in the system. The patient's remark can be used to store temporary information such as family health history and current medications/treatments patients are on.
 
-**Format**: `remark_p INDEX [r\REMARK]`
+**Format**: `remark_p INDEX r\REMARK`
```

Comment:
remark should be optional, when they don't input a remark, mediconnect will delete the pre-existing remark stored in the specified patient/appointment/doctor.

eg. remark_p 1 works, and returns this
<img width="757" alt="image" src="https://github.com/AY2324S1-CS2103T-T08-1/tp/assets/122232764/8d29eb2d-4335-4b8d-8792-6abf9fca1801">